listeners/steps/workflow_step.py

```diff
@@ -1,19 +1,21 @@
 import logging
-from typing import Optional, Union, List
+import os
+from typing import Optional, Union
 
-from slack_bolt import Ack, App
-from slack_bolt.workflows.step import WorkflowStep, Configure, Update, Complete, Fail
+from slack_bolt import Ack
+from slack_bolt.workflows.step import Configure, Update, Complete, Fail
 from slack_sdk import WebClient
 from slack_sdk.errors import SlackClientError
 
-from articles import fetch_articles, format_article
+from .articles import fetch_articles, format_article
 
 # Keys
 input_channel_ids = "channel_ids"
 input_query = "query"
 input_num_articles = "num_articles"
 
 logger = logging.getLogger(__name__)
+news_api_key = os.environ["NEWS_API_KEY"]
```

Comment:
Not sure if this is the best way to do this? 

Previously, the `news_api_key` was passed down to a function that registered the workflow with Bolt in `app.py` but this seems cleaner than having it to pass it through all the register functions found in the Python template but also odd because I feel like this should be in an initialize step.

Reply:
Good point. Considering the main feature of this app, I think having the API key in app.py makes more sense. You can refactor the code as I suggested above (`NewsFetcher`) plus modify `register_listeners` to relay both `app` and `news_fetcher` instances.